### PR TITLE
Document row filtering of the sys.shards table.

### PR DESCRIPTION
**Pull request by andreidan**

### CODE REVIEW

File: blackbox/docs/sql/system.txt [hunk 1]
```diff
@@ -898,6 +898,21 @@ The table schema is as follows:
...
+directly or inherited from the ``SCHEMA`` or ``CLUSTER`` (for more information 
+on privileges inheritance see :ref:`Hierarchical Inheritance of Privileges 
+<hierarchical_privileges_inheritance>`).
+However, being able to query ``sys.shards`` will not allow the user to retrieve 
+all the rows in the table as it can contain information related to tables over 
+which the connected user does not have any privileges. The only rows that will 
+be returned will be the ones the user is allowed to access.
+For example, if the user ``john`` has any privilege on the ``doc.books`` table 
+but no privilege at all on ``doc.locations``, when ``john`` issues a 
+``SELECT * from sys.shards`` statement, the shards information related to the
```
Comment (seut) [inline blackbox/docs/sql/system.txt]: minor: think either all or no keyword should be uppercase

File: blackbox/docs/sql/system.txt [hunk 2]
```diff
@@ -898,6 +898,21 @@ The table schema is as follows:
 |                      | table.                           |             |
 +----------------------+----------------------------------+-------------+
 
+Accessing the ``sys.shards`` table is subjected to the same privileges 
+constraints as the other tables, namely, in order to query it, the connected
```
Comment (nslater) [inline blackbox/docs/sql/system.txt]: new sentence at "namely"

File: blackbox/docs/sql/system.txt [hunk 3]
```diff
@@ -898,6 +898,21 @@ The table schema is as follows:
 |                      | table.                           |             |
 +----------------------+----------------------------------+-------------+
 
+Accessing the ``sys.shards`` table is subjected to the same privileges 
+constraints as the other tables, namely, in order to query it, the connected 
+user needs to have the ``DQL`` privilege on the ``sys.shards`` table, either 
+directly or inherited from the ``SCHEMA`` or ``CLUSTER`` (for more information
```
Comment (nslater) [inline blackbox/docs/sql/system.txt]: new sentence at "for more" and drop the parenthesis

File: blackbox/docs/sql/system.txt [hunk 4]
```diff
@@ -898,6 +898,21 @@ The table schema is as follows:
 |                      | table.                           |             |
 +----------------------+----------------------------------+-------------+
 
+Accessing the ``sys.shards`` table is subjected to the same privileges 
+constraints as the other tables, namely, in order to query it, the connected 
+user needs to have the ``DQL`` privilege on the ``sys.shards`` table, either 
+directly or inherited from the ``SCHEMA`` or ``CLUSTER`` (for more information 
+on privileges inheritance see :ref:`Hierarchical Inheritance of Privileges 
+<hierarchical_privileges_inheritance>`).
+However, being able to query ``sys.shards`` will not allow the user to retrieve
```
Comment (nslater) [inline blackbox/docs/sql/system.txt]: new para (extra newline)

File: blackbox/docs/sql/system.txt [hunk 5]
```diff
@@ -898,6 +898,21 @@ The table schema is as follows:
...
 +----------------------+----------------------------------+-------------+
 
+Accessing the ``sys.shards`` table is subjected to the same privileges 
+constraints as the other tables, namely, in order to query it, the connected 
+user needs to have the ``DQL`` privilege on the ``sys.shards`` table, either 
+directly or inherited from the ``SCHEMA`` or ``CLUSTER`` (for more information 
+on privileges inheritance see :ref:`Hierarchical Inheritance of Privileges 
+<hierarchical_privileges_inheritance>`).
+However, being able to query ``sys.shards`` will not allow the user to retrieve 
+all the rows in the table as it can contain information related to tables over
```
Comment (nslater) [inline blackbox/docs/sql/system.txt]: comma before "as it"

File: blackbox/docs/sql/system.txt [hunk 6]
```diff
@@ -898,6 +898,21 @@ The table schema is as follows:
...
+constraints as the other tables, namely, in order to query it, the connected 
+user needs to have the ``DQL`` privilege on the ``sys.shards`` table, either 
+directly or inherited from the ``SCHEMA`` or ``CLUSTER`` (for more information 
+on privileges inheritance see :ref:`Hierarchical Inheritance of Privileges 
+<hierarchical_privileges_inheritance>`).
+However, being able to query ``sys.shards`` will not allow the user to retrieve 
+all the rows in the table as it can contain information related to tables over 
+which the connected user does not have any privileges. The only rows that will 
+be returned will be the ones the user is allowed to access.
+For example, if the user ``john`` has any privilege on the ``doc.books`` table
```
Comment (nslater) [inline blackbox/docs/sql/system.txt]: new para at "for example" (extra newline)

File: blackbox/docs/sql/system.txt [hunk 7]
```diff
@@ -898,6 +898,24 @@ The table schema is as follows:
 |                      | table.                           |             |
 +----------------------+----------------------------------+-------------+
 
+Accessing the ``sys.shards`` table is subjected to the same privileges 
+constraints as the other tables. 
+Namely, in order to query it, the connected user needs to have the ``DQL``
```
Comment (nslater) [inline blackbox/docs/sql/system.txt]: "namely" should be on prev line

File: blackbox/docs/sql/system.txt [hunk 8]
```diff
@@ -898,6 +898,24 @@ The table schema is as follows:
 |                      | table.                           |             |
 +----------------------+----------------------------------+-------------+
 
+Accessing the ``sys.shards`` table is subjected to the same privileges 
+constraints as the other tables. 
+Namely, in order to query it, the connected user needs to have the ``DQL`` 
+privilege on the ``sys.shards`` table, either directly or inherited from the 
+``SCHEMA`` or ``CLUSTER``. 
+For more information on privileges inheritance see :ref:`Hierarchical
```
Comment (nslater) [inline blackbox/docs/sql/system.txt]: "For" should be on previous line. and actually sorry to change my mind, but let's put this in parenthesis again

**Pull request merged**